apps/phone/src/apps/twitter/components/AddTweetModal.tsx

```diff
@@ -80,8 +80,15 @@ const AddTweetModal = () => {
   const submitTweet = async () => {
     await promiseTimeout(200);
     const cleanedMessage = clean(message.trim());
-    if (cleanedMessage.length === 0) return;
-    if (!isValidMessage(cleanedMessage)) return;
+    if (cleanedMessage.length === 0) {
+      if (!allowNoMessage || !images || images.length === 0)
+      {
+        return
+      }
+    }
+
+    if (cleanedMessage.length > 0)
```

Comment:
Why did you remove the return here?

Reply:
I didn't, I modified it to to check to see if they have enabled the new setting in the twitter section of the config (disabled by default) to check if they wish to let users post only an image with no message. Otherwise it continues as normal. Not sure if I explained that correctly but hopefully you get the point

Reply:
shouldn't this just be `if (cleanedMessage.length > 0 && !isValidMessage(cleanedMessage)) return;` then?

The way this is formatted kinda just confuses me

Reply:
cleaned up